manual_scripts/active_directory/users.csv

```diff
@@ -12,3 +12,4 @@ eclaims,Password123,eclaims@dluhcdata.local,Eclaims,eclaims,"OU=Users,OU=dluhcda
 jasperreports,Password123,jasperreports@dluhcdata.local,Jasper Reports app,jasperreports,"OU=Users,OU=dluhcdata,DC=dluhcdata,DC=local"
 jasperuser,Password123,jasperuser@dluhcdata.local,Jasper User,jasperuser,"CN=Datamart,OU=Users,OU=dluhcdata,DC=dluhcdata,DC=local"
 jasperadminuser,Password123,jasperadminuser@dluhcdata.local,Jasper Admin User,jasperadminuser,"CN=Datamart,OU=Users,OU=dluhcdata,DC=dluhcdata,DC=local"
+keycloak,Password123,keycloak@dluhcdata.local,Keycloak app,keycloak,"OU=Users,OU=dluhcdata,DC=dluhcdata,DC=local"
```

Comment:
How does this get used?

Reply:
Oh I'd assumed they just signed with their own credentials to get their roles. I didn't see Password123 in the Delta PR. I'll leave a comment there on what I think's missing